Makefile

```diff
@@ -2,12 +2,29 @@ PACKAGE=github.com/openshift/cluster-node-tuning-operator
 PACKAGE_BIN=$(lastword $(subst /, ,$(PACKAGE)))
 PACKAGE_MAIN=$(PACKAGE)/cmd/$(PACKAGE_BIN)
 
+# By default we build the same architecture we are running
+# Override this by specifying a different GOARCH in your environment
+HOST_ARCH ?= $(shell uname -m)
+
+# Convert from uname format to GOARCH format
+ifeq ($(HOST_ARCH),aarch64)
+	HOST_ARCH=arm64
+endif
+ifeq ($(HOST_ARCH),x86_64)
+	HOST_ARCH=amd64
+endif
+
+# Define GOARCH as HOST_ARCH if not otherwise defined
+ifndef GOARCH
+	GOARCH=$(HOST_ARCH)
+endif
+
 # Build-specific variables
 OUT_DIR=_output
 GOBINDATA_BIN=$(OUT_DIR)/go-bindata
 BINDATA=pkg/manifests/bindata.go
 ASSETS=$(shell find assets -name \*.yaml)
-GO=GOOS=linux GO111MODULE=on GOFLAGS=-mod=vendor go
+GO=GOARCH=$(GOARCH) GOOS=linux GO111MODULE=on GOFLAGS=-mod=vendor go
```

Comment:
Is there a default when GOARCH is not explicitly specified ?

Reply:
Yep, GOARCH is defined as HOST_ARCH on line 19 above if GOARCH is otherwise not defined. Unless otherwise specificied GOARCH will match the current system